test/test_fail.cc

```diff
@@ -0,0 +1,16 @@
+#include "glog/logging.h"
+#include "gtest/gtest.h"
+
+void f(int i) {
+  if (i >= 10) {
+    CHECK(false);
+  } else {
+    f(i + 1);
+  }
+};
+
+// This file is just for demonstrating stacktrace on any failure.
+// Will be removed when merging!
```

Comment:
Should we have a non-removed test, e.g. a `EXPECT_DEATH` test that checks for a stack trace? https://github.com/YOU-i-Labs/googletest/blob/master/googletest/docs/V1_7_AdvancedGuide.md#death-tests